### DISCUSSION TITLE
More (small) marker and label sizes

### DISCUSSION THREAD

**Discussion by olinesn** [talmolab/sleap#1854] [Ideas]
Hello,

I'm writing to ask if an additional marker and font size options could be added? Most of the larger options are not very useful, and the steps at the low end are too big. Maybe add "9" between 6 and 12 for fonts, and a "3" in marker sizes between 2 and 4?

Thanks!
Stefan

**Comment by talmo** [comment]
Tracked in #1861 and implemented in #2057.

**Comment by roomrys** [comment]
Long awaited, but finally integrated. This issue has been fixed in the SLEAP 1.4.1 release!
